htdocs/compta/facture/class/facture.class.php

```diff
@@ -948,6 +948,7 @@ function createFromClone($socid=0)
 		$this->close_code         = '';
 		$this->close_note         = '';
 		$this->products = $this->lines;	// Tant que products encore utilise
+		$this->entity = $conf->entity;
```

Comment:
@atm-quentin 
it's not good if you share the invoices and you create a clone of a shared invoice from another entity!

Reply:
I did that because i tried to clone a propal (which is enable in the module multicompany) and it creates it in the current entity. So, is this an error on propal ?

Reply:
@atm-quentin 
if you share the invoices or others, an invoice of the entity 2 will be visible on the entity 1, if you clone an invoice of the entity 2 from the entity 1 it is necessary that the cloned invoice keeps the id of entity 2!